src/c-writer.cc

```diff
@@ -830,9 +831,9 @@ std::string CWriter::DefineGlobalScopeName(ModuleFieldType type,
 /* Names for params, locals, and stack vars are formatted as "w2c_" + name. */
```

Comment:
Let's update this comment too...

Reply:
Done in f4427d296c80dd2a44f5242ec656a4fdaa0732db

Reply:
Great, setting to auto-merge.